paper.tex

```diff
@@ -422,15 +428,34 @@ \subsection*{Scope of the specification}
 distribution of all the parameters, we wouldn't be doing simulations in the
 first place.}
 
-\begin{itemize}
-    \item Why we've avoided turning this into a model of everything, allowing
-    for the specification of genome properties like recombination rates,
-    etc.
-
-    \item Why we've versioned the specification, and how we've planned
-    for future extensions.
+Demes is meant to interface with a large number of simulation and inference
+methods, so we have restricted to the specification to only describe
+demographic features at the population level. As the underlying discrete
+demographic structure is the common thread connecting many widely used
+population genetic software, many features of underlying genome biology are
+omitted.  Most notably, mutation and recombination rates are left unspecified
+(though they can be encoded as metadata in the data structure). Genome ploidy
+is left ambiguous, and selection and dominance models are absent. Using a Demes
+model, which specifies deterministic population sizes, as the demographic input
+for simulation using software that allows for selection implies that selection
+is \emph{soft}, that is, genome fitnesses are rescaled relative to the mean
+population fitness \citep{christiansen1975hard}. It is possible to instead
+interpret population sizes as carrying capacities, allowing for a model of
+\emph{hard} selection. However, this should not be understood as the default
+interpretation of population sizes, which conventionally specifies a
+deterministic number of individuals.  Finally, as Demes assumes the
+exchangeability of individuals within populations, Demes does not provide a way
+to define geographic spatial structure within populations.
+
+Demes version 1.0 is intended to provide a stable and minimal specification
+that is compatible with a wide range of simulation softwares. Beginning with
+version 1.0, a Demes-specified model is expected to provide the version of
+Demes for which it is valid, and if unprovided, it should be a valid model
+under the default version, 1.0. Future extensions of the Demes specification
```

Comment:
I think this level of detail is for the spec really rather than the paper. We just need a sentence somewhere saying that the spec is versioned, and we commit to SemVer.